neurons/validator.py

```diff
@@ -965,23 +990,26 @@ def log_step(
                 "win_rate": win_rate[uid],
                 "win_total": wins[uid],
                 "weight": self.weights[uid].item(),
+                "norm_weight": sub_competition_weights[idx].item(),
```

Comment:
Optional: I find this name rather confusing. These are more "competition_weights", as you use in the table below. Both "weights" and these competition weights have been normalized, to some degree